bylaws.md

```diff
@@ -25,10 +25,9 @@
 
 ## Section 4. Vacancies
 1. When a vacancy arises during the middle of a term, the denhac Members will select by majority vote a replacement director to serve for the remainder of the departing director’s term. In the event of two or more directors departing, the incoming directors will fill the vacated terms. Directors with more votes will fill the vacancies with longer remaining terms respectively.
-2. Nominations for Directors to replace a Director whose term is ending for any reason may come from the Board or from the Members, at a meeting designated expressly for such purpose.
```

Comment:
Removed #2 - This was a holdover from when we had nominations and elections in person. The process for who can nominate is specified in Section 3, so this can be removed entirely.

`2. Nominations for Directors to replace a Director whose term is ending for any reason may come from the Board or from the Members, at a meeting designated expressly for such purpose.`